packages/admin/admin/src/form/ChipSelect.tsx

```diff
@@ -0,0 +1,163 @@
+import { ChevronDown } from "@comet/admin-icons";
+import { Chip, ChipProps, ComponentsOverrides, InputBase as MuiInputBase, InputBaseProps, MenuItem, Select, SelectChangeEvent } from "@mui/material";
+import { css, Theme, useThemeProps } from "@mui/material/styles";
+import * as React from "react";
+import { FormattedMessage } from "react-intl";
+
+import { createComponentSlot } from "../helpers/createComponentSlot";
+import { ThemedComponentBaseProps } from "../helpers/ThemedComponentBaseProps";
+
+export type ChipSelectClassKey = "root" | "inputBase" | "inputRoot" | "chip" | "select";
+
+const Root = createComponentSlot("div")<ChipSelectClassKey>({
+    componentName: "ChipSelect",
+    slotName: "root",
+})();
+
+const ChipInputRoot = createComponentSlot("div")<ChipSelectClassKey>({
+    componentName: "ChipSelect",
+    slotName: "inputRoot",
+})(
+    () => css`
+        display: flex;
+        flex-direction: column;
+        position: relative;
+    `,
+);
+
+const InputBase = createComponentSlot(MuiInputBase)<ChipSelectClassKey>({
+    componentName: "ChipSelect",
+    slotName: "inputBase",
+})(
+    () => css`
+        .MuiInputBase-input {
+            padding: 0;
+            height: 100% !important;
+            min-height: unset !important;
+        }
+    `,
+);
+
+export interface ChipSelectProps<T = string>
+    extends ThemedComponentBaseProps<{
+            root: "div";
+            inputRoot: "div";
+            select: typeof Select;
+            menuItem: typeof MenuItem;
+            inputBase: typeof MuiInputBase;
+        }>,
+        Omit<ChipProps, "children" | "onChange"> {
+    children?: React.ReactNode;
+    getOptionLabel?: (option: T) => string;
+    getOptionValue?: (option: T) => string;
+    options?: T[];
+    selectedOption?: T;
+    onChange: (event: SelectChangeEvent) => void;
+    fullWidth?: boolean;
+}
+
+const ChipInput = ({
+    chipProps,
+    inputBaseProps,
+    inputRootProps,
+    ...restProps
+}: InputBaseProps & {
+    chipProps?: Omit<ChipProps, "children">;
+    inputBaseProps?: InputBaseProps;
+    inputRootProps?: React.ComponentPropsWithoutRef<"div">;
+}) => {
+    const fallBackLabel = <FormattedMessage id="comet.admin.form.chipselect.placeholder" defaultMessage="Select a value" />;
+
+    return (
+        <ChipInputRoot {...inputRootProps}>
+            <Chip
```

Comment:
In the design the content of the `Chip` is aligned to the left, it is centered in the implementation.
_(When using `fullWidth`)_

| Design | Implementation |
| --- | --- |
| <img width="308" alt="Screenshot 2024-06-20 at 12 09 12" src="https://github.com/vivid-planet/comet/assets/6264317/06b9d917-1272-4d97-aa9c-3c559dc276ed"> | <img width="267" alt="Chip Select Implementation" src="https://github.com/vivid-planet/comet/assets/6264317/d8ec49a4-b6d7-4f2f-84cd-7879739a363b"> |